clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -822,8 +822,9 @@ def NSobjectAttribute : DiagGroup<"NSObject-attribute">;
 def NSConsumedMismatch : DiagGroup<"nsconsumed-mismatch">;
 def NSReturnsMismatch : DiagGroup<"nsreturns-mismatch">;
 
+def UnknownAttributeNamespace : DiagGroup<"unknown-attribute-namespace">;
```

Comment:
Do we want to have a slightly different diagnostic wording for this case? e.g., `unknown attribute namespace '%0'; attribute '%0::%1' ignored`?

That way it's more clear that we don't know any attributes from that namespace?

Also, given that it's `-Wunknown-attributes` (plural) should this be `-Wunknown-attribute-namespaces`? I don't have a strong opinion, just caught my attention that one was plural and the other wasn't.